src/obstacles_2d_module.py

```diff
@@ -71,6 +76,7 @@ def get_attribute_from_config(attribute_name:str,  default):
         ransac_stop_probability = get_attribute_from_config("ransac_stop_probability", 0.99)
         prism_z_dim = get_attribute_from_config('obstacles_height_mm', 1.0)
         self.normal_vector = get_attribute_from_config("normal_vector", 'z')
+        self.min_range_mm = get_attribute_from_config("min_range_mm", None, float)
```

Comment:
I think we can remove the of_type field and instead default this to 0. If the min_range filter is 0, then no points will ever be filtered since sqrt(x^2)+y^2) cannot be less than 0

Reply:
I rather not go through the filtering loop at all if `self.min_range_mm` is default

Reply:
Fair point!